src/main/java/core/basesyntax/dao/CsvFileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.dao;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class CsvFileWriterImpl implements CsvFileWriter {
+
+    @Override
+    public void write(String data, String filePath) {
+        try {
+            Files.write(Paths.get(filePath),data.getBytes());
```

Comment:
Consider adding a space after the comma in `data.getBytes()` for better readability. It should be `Files.write(Paths.get(filePath), data.getBytes());`.